pageserver/src/disk_usage_eviction_task.rs

```diff
@@ -570,7 +570,7 @@ async fn collect_eviction_candidates(
         tenant_candidates
             .sort_unstable_by_key(|(_, layer_info)| std::cmp::Reverse(layer_info.last_activity_ts));
         let mut cumsum: i128 = 0;
-        for (timeline, layer_info) in tenant_candidates.into_iter() {
+        for (timeline, layer_info) in tenant_candidates {
```

Comment:
I guess this is the root cause for the `&Arc` changes?

Reply:
The line here does exactly the same, the `into_iter` calling is extra.

Reply:
Dropped this, let's have it on the large PR.